From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@sury.org>
Date: Tue, 13 Nov 2018 08:21:31 +0000
Subject: Don't use sed found by configure,
 use the sed command as available in the host system

---
 scripts/phpize.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/phpize.in b/scripts/phpize.in
index 2f468ee..db03fbe 100644
--- a/scripts/phpize.in
+++ b/scripts/phpize.in
@@ -7,7 +7,7 @@ exec_prefix="`eval echo @exec_prefix@`"
 phpdir="$prefix/lib/php/@DEBIAN_PHP_API@/build"
 includedir="$prefix/include/php/@DEBIAN_PHP_API@"
 builddir="`pwd`"
-SED="@SED@"
+SED="sed"
 
 aclocaldir="$prefix/share/aclocal"
 FILES_BUILD="mkdep.awk scan_makefile_in.awk shtool ax_check_compile_flag.m4 ax_gcc_func_attribute.m4"
